src/main/webapp/creatives/maps/UserActivityLogger.js

```diff
@@ -1,45 +1,58 @@
 
-import {generateSessionId, URL} from './utils.js';
+import {generateSessionId} from './utils.js';
+import {Urls} from '../constants.js';
 
-class UserActivityLogger {
+export default class UserActivityLogger {
 
   constructor() {
     /**
      * Each instance gets a correlator to uniquely identify it.
      * @private @const {string}
      */
     this.correlator_ = generateSessionId();
+
+    /** @private @const {string} */
```

Comment:
This variable might benefit from some description here but if you think baseUrl is descriptive enough then it's good as it